src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -334,22 +334,26 @@ public static void makeGenotypeCall(final int ploidy,
                     gb.alleles(bestMatchToOriginalGT(allelesToUse, originalGT.getAlleles()));
                 }
             } else {
-                final int maxLikelihoodIndex = MathUtils.maxElementIndex(genotypeLikelihoods);
-                final GenotypeAlleleCounts alleleCounts = GenotypesCache.get(ploidy, maxLikelihoodIndex);
-                final List<Allele> finalAlleles = alleleCounts.asAlleleList(allelesToUse);
-                final double gq = GenotypeLikelihoods.getGQLog10FromLikelihoods(maxLikelihoodIndex, genotypeLikelihoods);
-                if (finalAlleles.contains(Allele.NON_REF_ALLELE)) {
-                    final Allele ref = allelesToUse.stream().filter(Allele::isReference).collect(Collectors.toList()).get(0);
-                    gb.alleles(Collections.nCopies(ploidy, ref));
-                    gb.PL(new int[genotypeLikelihoods.length]).log10PError(0);
-                } else if (maxLikelihoodIndex == 0 && gq > GATKVariantContextUtils.SUM_GL_THRESH_NOCALL){ //GLs are log10s, so they're negative, GQ is also negative, so less negative is greater than
-                    gb.alleles(Collections.nCopies(ploidy, Allele.NO_CALL));
+                if (genotypeLikelihoods == null) {
```

Comment:
the diff here got aggressive, but all I did was add this if block and move the rest of the code in the else block into another else block